js/main.js

```diff
@@ -102,6 +134,24 @@ function switchViews(viewName) {
   data.view = viewName;
 }
 
+$entryList.addEventListener('click', function clickEdit(event) {
+  if (!event.target.matches('i')) {
+    return;
+  }
+  switchViews('entry-form');
+  for (var i = 0; i < data.entries.length; i++) {
+    if (JSON.stringify(data.entries[i].entryId) === (event.target.closest('li').getAttribute('data-entry-id'))) {
+      data.edit = data.entries[i];
```

Comment:
Do you mean `data.editing` in this function? There isn't a `data.edit` property we've prepared for you to use.